railties/test/application/console_test.rb

```diff
@@ -23,6 +26,7 @@ class Post < ActiveRecord::Base
 
   def teardown
     teardown_app
+    FileUtils.remove_entry(@home_tmp_dir) if @home_tmp_dir
```

Comment:
Maybe we should move the tmpdir creation to `spawn_console`, or remove the `if` condition here, since it always gets created via `setup`?

Reply:
@carlosantoniodasilva Done, thanks for the review!